sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -105,23 +109,29 @@ class ParquetFilters(
       fieldType: ParquetSchemaType)
 
   private case class ParquetSchemaType(
-      originalType: OriginalType,
+      logicalTypeAnnotation: LogicalTypeAnnotation,
       primitiveTypeName: PrimitiveTypeName,
       length: Int,
-      decimalMetadata: DecimalMetadata)
+      decimalLogicalType: DecimalLogicalTypeAnnotation)
```

Comment:
this may no longer needed since we can just use `logicalTypeAnnotation`?

Reply:
9a7ec8c try to remove this from `ParquetSchemaType`